GNUmakefile

```diff
@@ -0,0 +1,116 @@
+UNAME := $(shell uname)
```

Comment:
Can we put this somewhere other than the top-level, given that it really only exists to deal with the new bindist workflow?

Reply:
I'd prefer not to... otherwise the Makefile has to find the repository root or CI has to copy it back. It really belongs to the top level dir.